### DISCUSSION TITLE
Climate e SmartIr

### DISCUSSION THREAD

**Discussion by giandena** [indomus/forum#2069] [HUB: Home Assistant]
Buongiorno,
da poco sono approdato ad Home Assistant che ho installato su un Rpi2.
Ho enormi difficoltà a comandare il mio condizionatore Panasonic con il telecomando Broadlink Rm3mini.
Il dispositivo è correttamente integrato in HA (2022.11.1), tant'è che se utilizzo i servizi remote_learn command e remote_send command il dispositivo Rm3mini apprende ed invia codici.
Sotto HACS ho installato SmartIr 1.17.6 come repository personalizzata.
Nella directory /home/pi/homeassistant/custom_components/smartir/codes/climate vedo il file 1023.json che viene scaricato in automatico dal file configuration.yaml, ma il problema è che quando integro nella plancia il dispositivo termostato associato al condizionatore, pur vedendo regolarmente il dispositivo, ad ogni pressione dei comandi il Broadlink resta assolutamente spento.
Solo dopo aver premuto i tasti di gestione del condizionatore nella plancia nel file registro di HA appare questo errore:

Logger: homeassistant.helpers.service
Source: helpers/service.py:637
First occurred: 10:54:12 (2 occurrences)
Last logged: 10:54:15
Unable to find referenced entities remote.rm3sala_remote or it is/they are currently not available

che ovviamente a me sembra disponibile!

Ho provato e letto tanti documenti al riguardo dovrebbe essere una cosa alquanto semplice ed invece proprio non riesco. 
Qualcuno di buona volontà ha la pazienda di aiutarmi a capire dove sbaglio?

Allego il file configuration.yaml sperando di aver correttamente utilizzato Markdown 

`
# Loads default set of integrations. Do not remove.
default_config:

# Load frontend themes from the themes folder
frontend:
  themes: !include_dir_merge_named themes

# Text to speech
tts:
  - platform: google_translate

automation: !include automations.yaml
script: !include scripts.yaml
scene: !include scenes.yaml

smartir:

climate:
  - platform: smartir
    name: Sala AC
    unique_id: sala_ac
    device_code: 1023
    controller_data: remote.rm3sala_remote
    #temperature_sensor: sensor.temperature
    #humidity_sensor: sensor.humidity
    #power_sensor: binary_sensor.ac_power
`

**Comment by Gigiiacu** [comment]
Before the update everything worked for months. Now from this error. Who helps me?
from hacs i can't even remove it (uninstall it)

Logger: homeassistant.components.hassio
Source: components/hassio/__init__.py:518
Integration: Home Assistant Supervisor ([documentation](https://www.home-assistant.io/integrations/hassio), [issues](https://github.com/home-assistant/home-assistant/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+hassio%22))
First occurred: 10:50:55 (2 occurrences)
Last logged: 14:12:43

The system cannot restart because the configuration is not valid: Integration error: smartir - Requirements for smartir not found: ['aiofiles==0.6.0']. Platform error climate.smartir - Requirements for smartir not found: ['aiofiles==0.6.0']. Platform error climate.smartir - Requirements for smartir not found: ['aiofiles==0.6.0'].

**Reply by Gigiiacu** [reply]
configuration.yaml
smartir:
climate: !include climate.yaml

climate.yaml
  - platform: smartir
    name: Ferroli
    unique_id: Ferroli_ac
    device_code: 1180
    controller_data: remote.eremote2_remote
    temperature_sensor: sensor.temperature_Ferroli
    humidity_sensor: sensor.humidity_Ferroli
    power_sensor: binary_sensor.stato_operativo_clima_cucina
    
  - platform: smartir
    name: Sanyo
    unique_id: Sanyo_ac
    device_code: 1501
    controller_data: remote.eremote_remote
    temperature_sensor: sensor.temperature_Sanyo
    humidity_sensor: sensor.humidity_Sanyo
    power_sensor: binary_sensor.stato_operativo_clima_camere
